### ISSUE TITLE
Error message when clicking on "Release Notes" button in update message.

### ISSUE THREAD

**Issue by BobbyBabes** [Microsoft/vscode#17455]
**- VSCode Version:**

```
Version 1.7.2
Commit 7ba55c5860b152d999dda59393ca3ebeb1b5c85f
Date 2016-11-21T22:14:18.217Z
Shell 1.3.8
Renderer 52.0.2743.82
Node 6.5.0
```
**- OS Version:**
Arch Linux.
``Linux designbox 4.8.13-1-ARCH #1 SMP PREEMPT Fri Dec 9 07:24:34 CET 2016 x86_64 GNU/Linux``

Steps to Reproduce:

1. Start Visual Studio Code.
2. Wait for the info message "There is an available update" to appear.
3. Click the "Release Notes" button.
4. Bam! Head shot. :) ``Unable to open 'Release Notes: 38746938a4ab94f2f57d9e1309c51fd6fb37553d': Cannot read property 'replace' of null``

![vscode-update-release-notes-error-1](https://cloud.githubusercontent.com/assets/22870070/21286278/bac6d5a2-c44f-11e6-8fec-9236704db676.png)

![vscode-update-release-notes-error-2](https://cloud.githubusercontent.com/assets/22870070/21286281/c4e2bd4e-c44f-11e6-8b0a-f21a4de570db.png)

**Comment by bpasero** [comment]
I can reproduce and it looks like the commit version ID ends up into the check for version [here](https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/parts/update/electron-browser/update.ts#L53) and that just makes the function return `null` as content for release notes.

I think this was caused by https://github.com/Microsoft/vscode/commit/d7e4046546a924e20c1e4a3d9b5bce497868ec49 where instead of the product version we return the commit version. 

Not clear to me why we did this change though.

**Comment by joaomoreno** [comment]
This is fixed in 1.8.1.

**Comment by BobbyBabes** [comment]
Thanks!
Now there is no update to confirm though. ;)
